src/js/common/polyfills.js

```diff
@@ -0,0 +1,26 @@
+// Common Javascript environment setup to be used by all entrypoints.
+//
+// Keep this short. It should mostly be polyfills, global site style, and any
+// truly pervasive libraries. Be careful if you are tempted to put jquery or
+// other such libraries in here. Most of the site does not use these legacy
+// frameworks and it belongs in a lower-level module.
+
+require('babel-polyfill');
```

Comment:
So we still need to use `babel-polyfill`, but now that we're using `babel-preset-env` it's only loading the polyfills we actually need based on the environments we support?